examples/yacat/yacat.py

```diff
@@ -78,7 +78,7 @@ async def worker_find_password(ctx: WorkContext, tasks):
 
             output_file = f"hashcat_{skip}.potfile"
             ctx.download_file(f"/golem/work/hashcat_{skip}.potfile", output_file)
-            yield ctx.commit()
+            yield ctx.commit(timeout=timedelta(minutes=25))
```

Comment:
I'm not sure it makes sense to set the timeout that long. Timeouts are used to ensure the task that is blocked on one provider for a long time can be stopped and sent to another provider. In this case, after 25 minutes it may be to late to send the task to another provider before the whole computation times out.

Reply:
you are right. 10 minutes would be better